### PR TITLE
Fixed the css bug in edit name functionality

### PR DESCRIPTION
**Pull request by Dhamodhar-DDR**
## Issue that this pull request solves

 Closes: #240

## Proposed changes
Rename CSS issue in responsive screens.  All the buttons in rename functionality should be in one single row

### Brief description of what is fixed or changed
The input tag, save button and the cancel button will now be in one single row.
The rename functionality will not be able to take null/void names.

## Types of changes

_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] My changes does not break the current system and it passes all the current test cases.

## Screenshots
Screenshot from 2020-12-12 08-52-09
Screenshot from 2020-12-12 08-51-31

## Other information

Any other information that is important to this pull request

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -60,9 +60,9 @@
     "typescript": "~3.5.3"
   },
   "husky": {
-  "hooks": {
-    "pre-commit": "ng lint",
-    "pre-push":"ng test --no-watch"
+    "hooks": {
```
Comment (Kajol-Kumari) [inline package.json:63]: Hey @Dhamodhar-DDR why do we need this change in here? Please revert this change

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
imageedit_3_2629519185
Hey @Dhamodhar-DDR please make the suggested changes and also provide a bit of spacing between the input field and the button as marked in the image above.

File: package.json (hunk 1 above)
Comment (Dhamodhar-DDR) [reply package.json:63]: I dont think there is any change here, i just added a tab space to those three lines.
